app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
With the latest patch I am working on right now it will actually store VMs that comprise catalog_items. The hierarchy of catalogs is like this:

```organization -> catalogs -> catalog_items -> vapp_template -> vms```

To my understanding the VMs here are actual images (some metadata + the actual image in whatever format was used when uploading the template). The idea is that for the provisioning users will select actual (neither vapp templates nor catalog items) and the backend is going to compose vapps on the fly.

@cankarm Is there something I actually missed here?